opm/simulators/linalg/MILU.cpp

```diff
@@ -255,11 +255,11 @@ void milun_decomposition(const M& A, int n, MILU_VARIANT milu, M& ILU,
 }
 
 
-template double Opm::detail::identityFunctor(const double&);
-template double Opm::detail::oneFunctor(const double&);
-template double Opm::detail::signFunctor(const double&);
-template double Opm::detail::isPositiveFunctor(const double&);
-template double Opm::detail::absFunctor(const double&);
+  template std::function<double(const double&)> identityFunctor;
+  template std::function<double(const double&)> oneFunctor;
+  template std::function<double(const double&)> signFunctor;
+  template std::function<double(const double&)> isPositiveFunctor;
+  template std::function<double(const double&)> absFunctor;
```

Comment:
Seemed correct before

Reply:
So even if the functions are declared in `Opm::detail `as **non-static** functions, they can/must be called with the `Opm::detail:: `in front. 
I thought that was only for **static member functions** and "the **non-static**" ones should be called like `myFuction<myClass::mySubClass><type_if_it_comes_from_a_template>`.

I was wrong :/